rules/TypeDeclaration/Rector/Property/TypedPropertyFromStrictConstructorRector.php

```diff
@@ -197,4 +201,25 @@ private function shouldSkipPropertyType(Type $propertyType): bool
 
         return $this->isDoctrineCollectionType($propertyType);
     }
+
+    private function skipPublicProperty(Node\Stmt\Property $property, ClassReflection $classReflection, Scope $scope): bool
+    {
+        if ($property->isPublic()) {
+            $isReadOnlyByPhpdoc = false;
```

Comment:
we only care about readonly phpdoc, because native readonly classes cannot have non-typed-properties by definition